### PR TITLE
chore(circuits): integrate linter

### PR DESCRIPTION
**Pull request by 0xmad**
# Description

- [x] Apply linter and type fixes
- [x] Dependency management

## Additional Notes

N/A

## Related issue(s)

Related to #845 

## Confirmation

- [x] I have read and understand MACI's [contributor guidelines](https://github.com/privacy-scaling-explorations/maci/blob/dev/CONTRIBUTING.md) and [code of conduct](https://github.com/privacy-scaling-explorations/maci/blob/dev/CODE_OF_CONDUCT.md).
- [x] I have read and understand MACI's [GitHub processes](https://github.com/privacy-scaling-explorations/maci/discussions/847).
- [x] I have read and understand MACI's [testing guide](https://maci.pse.dev/docs/testing).

### CODE REVIEW

File: circuits/ts/__tests__/CeremonyParams.test.ts [hunk 1]
```diff
@@ -186,14 +188,14 @@ describe("Ceremony param tests", function () {
       });
     });
 
-    describe("TallyVotes circuit", function () {
+    describe("TallyVotes circuit", function test() {
       this.timeout(900000);
 
-      let circuit: tester.WasmTester;
+      let testCircuit: tester.WasmTester;
```
Comment (ctrlc03) [inline circuits/ts/__tests__/CeremonyParams.test.ts:194]: Saw you changed here but not in other files, let's keep them all the same?

**Review by ctrlc03 — APPROVED** [review]
Looks good, thank you! Just left one comment but nothing stopping this from being merged

File: circuits/ts/__tests__/CeremonyParams.test.ts (hunk 1 above)
Comment (0xmad) [reply circuits/ts/__tests__/CeremonyParams.test.ts:194]: I changed this one because it conflicts with `circuit` const which is declared above.

Comment (ctrlc03) [reply circuits/ts/__tests__/CeremonyParams.test.ts:194]: Oh true

**Pull request merged**